packages/components/src/public/QueryInfo.ts

```diff
@@ -3,9 +3,17 @@ import { List, Map, OrderedMap, Record } from 'immutable';
 
 import { Filter } from '@labkey/api';
 
-import { insertColumnFilter, LastActionStatus, QueryColumn, QuerySort, SchemaQuery, ViewInfo } from '..';
+import {
+    insertColumnFilter,
+    LastActionStatus,
+    QueryColumn,
+    QuerySort,
+    SchemaQuery,
+    ViewInfo
+} from '..';
 
 import { toLowerSafe } from '../internal/util/utils';
+import { STORAGE_UNIQUE_ID_CONCEPT_URI } from '../internal/components/domainproperties/constants';
```

Comment:
is `STORAGE_UNIQUE_ID_CONCEPT_URI` used here, seems like it might be an unused import.